### ISSUE TITLE
Add job to trigger a new deployment on demand

### ISSUE THREAD

**Issue by sopel** [cityindex/logsearch#301]
This has been extracted from #24 and depends on #300:

* this should deploy the artifacts uploaded via #300 to a new (test) environment
 * it's supposed to provide _push-button deployments of any version of the software to any environment on demand_ (see [Continuous Delivery](http://martinfowler.com/bliki/ContinuousDelivery.html)) down the road
* for the time being the job should be only be triggered on demand for testing purposes, i.e. continuous deployment and update of production environments are out of scope here

**Comment by sopel** [comment]
Closed as _Won't Fix_ due to project being retired to the [CityIndex Attic](https://github.com/orgs/cityindex-attic).
